source/_references/sourceimages.md

```diff
@@ -141,6 +141,20 @@ $isDeleted = $client->deleteSourceImagesWithBinaryHash('03b3e8a0bdd76ef55c021066
 var_dump($isDeleted);
 ```
 
+## Restore a source image
+
+To restore a deleted image,  do a POST on the `/sourceimages` route with providing the identifying hash and appending '/restore'.
```

Comment:
instead of describing the method and URL, how about doing:
> `POST /sourceimages/{hash}/restore` restores a deleted image.

Or something similar to that?